### PR TITLE
[5.5] Github Issue Template: add commented preamble about alternatives to get help

### PR DESCRIPTION
**Pull request by mfn**
As the number of general support requests is still high and cause quite some noise, try with a more prominent header when creating a new issue.

If accepted, should be backported to at least 5.5

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+<!--
+Please use this issue tracker only for reporting bugs in Laravel.
+
+If you have question/seek support, please ask on the forums:
+- Uou can use https://laracasts.com/discuss or https://laravel.io/forum which are forums with a very large community of developers helping each other.
```
Comment (m1guelpf) [inline .github/ISSUE_TEMPLATE.md]: Typo here

Comment (m1guelpf) [inline .github/ISSUE_TEMPLATE.md]: Maybe we should also link to StackOverflow...

Comment (mfn) [reply .github/ISSUE_TEMPLATE.md]: 👍 fixed

**Comment by GrahamCampbell** [comment]
Changed the target from master (5.7) to 5.5.

**Comment by GrahamCampbell** [comment]
> If accepted, should be backported to at least 5.5

We typically don't work like that. We merge upwards. :)

**Comment by m1guelpf** [comment]
@GrahamCampbell When was 5.6 released?

**Comment by GrahamCampbell** [comment]
5.6 is not scheduled to be released until February.

**Comment by m1guelpf** [comment]
@GrahamCampbell Then, why is there a `5.6` branch? I thougth that branch was only created on release time...

**Comment by GrahamCampbell** [comment]
> I thougth that branch was only created on release time

Nobody said that?

**Comment by mfn** [comment]
> We typically don't work like that. We merge upwards. :)

Ops, sorry ;) Thanks!

**Comment by laurencei** [comment]
> Nobody said that?

@GrahamCampbell - It's in the docs:

https://laravel.com/docs/5.5/contributions#which-branch

> Major new features should always be sent to the master branch, _which contains the upcoming Laravel release_.

So do we target `5.6` or `master` now with PRs we want in 5.6?

**Comment by GrahamCampbell** [comment]
The docs are wrong in this instance, since there are two branches now, 5.6 and master.

**Comment by GrahamCampbell** [comment]
Well, the docs are right, master is for an upcoming release, just not for 5.6.

**Comment by GrahamCampbell** [comment]
Once we get closer to 5.6, I imagine master will be where breaking things go, instead of to 5.6.

**Comment by laurencei** [comment]
...so where do we target PRs to? 5.6 or master?

**Comment by GrahamCampbell** [comment]
5.6 for now

**Pull request merged**